### ISSUE TITLE
Kong Manager Portal Editor - unable to delete folder

### ISSUE THREAD

**Issue by meesvandongen** [Kong/kong#8535]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Kong version (`$ kong version`)

2.7.1.2

### Current Behavior

When using the editor, right clicking and selecting delete folder brings up a pop-up. Clicking [Proceed] in this pop-up does not do anything. No network requests are made; and the pop-up remains open.

### Expected Behavior

The folder is deleted.

### Steps To Reproduce

```markdown
1. Make a workspace
2. Enable Portal
3. Go to Portal Editor
4. Right click on folder
5. choose Delete folder
6. Choose proceed
```

**Comment by mayocream** [comment]
Hi @meesvandongen, thanks for reporting this issue.

I've created an internal ticket to track the process to solve the problem :)
We are trying to reprod and solve that ASAP.

**Comment by mayocream** [comment]
The bugfix was included in the 2.8.1.0 release.
